site/specs-temp/numbers.json

```diff
@@ -7515,6 +7515,292 @@
         }
       }
     },
+    "/accounts/{accountId}/importTnOrders/voice": {
+      "get": {
+        "tags": [
+          "/accounts"
+        ],
+        "description": "Retrieves the ImportVoiceTnOrder requests for the given account ID. A maximum of 1,000 orders can be retrieved per request. If no date range or specific query parameter (marked by <b class=\"required\">*</b> below) is provided, the order results will be limited to the last two years.\n",
+        "summary": "List Import Voice Tn orders",
+        "operationId": "ReadImportVoiceTnOrders",
+        "parameters": [
+          {
+            "name": "accountId",
```

Comment:
`accountId`, `orderId` and `status` should be defined as reusable `components`. The `status` should be named accordingly and defined as `enum`

Reply:
fixed

Reply:
@bpateldx plz re-review